gitversion.yml

```diff
@@ -2,8 +2,9 @@ next-version: 9.6.0
 commit-date-format: 'yyyyMMdd'
 assembly-file-versioning-format: '{Major}.{Minor}.{Patch}.{CommitsSinceVersionSource}'
 mode: ContinuousDeployment
+ignore:
+  commits-before: 2019-01-01T00:00:00
```

Comment:
Should this be 2020-01-01?

Reply:
Hmm, yeah, this guys did a mistake, let me run a try and change that date...